### PR TITLE
(#1) Add support for Github login

### PR DESCRIPTION
**Pull request by oriolpiera**
## Description

Fix: https://github.com/GeeksCAT/anem-per-feina/issues/1

This PR adds a new feature, login with your Github account.

image

## Environment variables

- `SOCIAL_AUTH_GITHUB_KEY`: Key of Githbu third-party login App
- `SOCIAL_AUTH_GITHUB_SECRET`: Secret of Github third-party login App

## Code formating

The code has to be well formatted following the formatting rules. For example in Python the code has to follow the PEP8.

Before create your PR, please make sure your code is well formatted and styled doing:
```bash
$ >> pre-commit run --all
```

### CODE REVIEW

File: .env.dev.sample [hunk 1]
```diff
@@ -13,3 +13,7 @@ POSTGRES_PORT=5432
 
 # Sentry
 SENTRY_URL='https://publickey@ingest.sentry.io/5'
+
+# Third-party login keys
+SOCIAL_AUTH_GITHUB_KEY=githubt-key
```
[the remark this answers is not in the document]
Comment (jbagot) [reply .env.dev.sample:18]: If they are not mandatory I wouldn't add them here

**Review by jbagot — CHANGES_REQUESTED** [review]

File: jobs/settings.py [hunk 2]
```diff
@@ -34,6 +34,8 @@
     "accounts",
     "constance",
     "notifications",
+    "constance.backends.database",
```
[the remark this answers is not in the document]
Comment (oriolpiera) [reply jobs/settings.py]: done

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,6 @@
 asgiref==3.2.3
+# celery may commit on different PR
+celery==5.0.1
```
[the remark this answers is not in the document]
Comment (oriolpiera) [reply requirements.txt]: done

File: .env.dev.sample (hunk 1 above)
[the remark this answers is not in the document]
Comment (oriolpiera) [reply .env.dev.sample]: removed, done.

**Review by francescarpi — APPROVED** [review]

**Comment by francescarpi** [comment]
Nice job!

**Pull request merged**